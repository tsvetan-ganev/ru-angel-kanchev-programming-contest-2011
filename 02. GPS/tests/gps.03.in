40

Debelets
Devin
Devnja
Djebel
Dimitrovgrad
Dimovo
Dobrinishte
Dobrich
Dolni chiflik
Dolna banja
Dolna Mitropolija
Dolna Orjahovitsa
Dolni Dabnik
Dospat
Dragoman
Drjanovo
Dulovo
Dunavtsi
Dupnitsa
Dalgopol
Elena
Elin Pelin
Elhovo
Etropole
Zavet
Zemen
Zlataritsa
Zlatitsa
Zlatograd
Ivajlovgrad
Ignatievo
Iskar
Isperih
Ihtiman
Kableshkovo
Kavarna
Kazanlak
Kalofer
Kamen
Kameno
