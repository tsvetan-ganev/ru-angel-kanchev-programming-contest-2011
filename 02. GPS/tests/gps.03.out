d e z i k